.github/workflows/cypress

```diff
@@ -0,0 +1,17 @@
+name: Cypress Tests
```

Comment:
According to [the doc](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#about-yaml-syntax-for-workflows), I think we need to make the file a .yaml one.